src/pages/HomePage/Homepage.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {Link} from "react-router-dom";
+import "./Homepage.css";
+
+const Homepage = () => {
+    return (
+        <section className="home-page-wrapper">
+            <div className="home-page-top">
+                <h2>
+                    <span className='home-page-title'>NOTE </span>IT
+                </h2>
+            </div>
+            <div className="home-content-container">
+                <div className="home-content">
+                    <span></span>
```

Comment:
you can remove this span right?

Reply:
It is for this 
![image](https://user-images.githubusercontent.com/76832165/160265440-ca994f53-6342-4141-992a-618e3e4121de.png)